content/docs/lists-and-keys.md

```diff
@@ -237,18 +237,18 @@ function Blog(props) {
 }
 
 const posts = [
-  {id: 1, title: 'Hello World', content: 'Welcome to learning React!'},
-  {id: 2, title: 'Installation', content: 'You can install React from npm.'}
+  {id: 1, title: ‘שלום עולם’, content: ‘!React ברוכים הבאים ללמידת’},
+  {id: 2, title: ‘התקנה’, content: ’npm-מ React אתה יכול להתקין את’}
```

Comment:
Changing the quotes might break the code

Reply:
Why do you think it might break the code?

Reply:
If it is a backtick ``` it is okay, but if it is another type of quote, which I think it is, it will not work if someone copies and pastes it in their editor, since it is not a legal quote char in ES